code/modules/lighting/lighting_object.dm

```diff
@@ -8,6 +8,9 @@
 	///the turf that our light is applied to
 	var/turf/affected_turf
 
+	/// Area which gets linked to a lighting object to make it consider the luminosity from the day/night blending from the area. Yes this isn't ideal, but applying luminosity up to 2 (from both sources) on the turf is not ideal either
+	var/area/day_night_area
+
```

Comment:
ya wot mate

Reply:
It's required as in testing, the adjacent turfs can sometimes be overriden by a different lightsource, and just adding to the luminosity would be stupid as the turf would not be able to be set to 0 which can happen sometimes.